packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/SignTransactionsStep/SignTransactionsStep.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
+import CWPageLayout from 'views/components/component_kit/new_designs/CWPageLayout';
+
+import { LaunchContestStep } from '../../types';
+
+import './SignTransactionsStep.scss';
+
+interface SignTransactionsStepProps {
+  onSetLaunchContestStep: (step: LaunchContestStep) => void;
+}
+
+const SignTransactionsStep = ({
+  onSetLaunchContestStep,
+}: SignTransactionsStepProps) => {
+  return (
+    <CWPageLayout>
+      <div className="SignTransactionsStep">
+        sign transactions
```

Comment:
should this be `Sign Transactions`?

Reply:
it's just a placeholder, will be changed anyways